cypress/e2e/awx/resources/projects.cy.ts

```diff
@@ -480,97 +583,119 @@ describe('Projects', () => {
       cy.clickTab(/^Job templates$/, true);
     });
 
-    it.skip('can associate a project with a newly created job template and view that JT on the templates tab of the project', () => {
-      //use the project created in the beforeEach block
-      //create a JT in this test and specifically associate the project to it
-      //visit the templates tab of the project and assert the JT showing there
-      //delete the JT at the end of this test
+    it('can associate a project with a newly created job template and view that JT on the templates tab of the project', function () {
+      cy.createAwxProject({ organization: (this.globalOrganization as Organization).id }).then(
+        (thisProject) => {
+          cy.navigateTo('awx', 'templates');
+          cy.filterTableByMultiSelect('name', [jobTemplate.name]);
+          cy.getTableRow('name', jobTemplate.name, { disableFilter: true }).should('be.visible');
+          cy.selectTableRow(jobTemplate.name, false);
+          cy.getBy('[data-cy="edit-template"]').click();
+          cy.verifyPageTitle('Edit Job Template');
+          cy.selectDropdownOptionByResourceName('project', thisProject.name);
+          cy.intercept('PATCH', awxAPI`/job_templates/${jobTemplate.id.toString()}/`).as('edited');
+          cy.getByDataCy('Submit').click();
+          cy.wait('@edited')
+            .its('response.body')
+            .then((editedJt: JobTemplate) => {
+              expect(editedJt.project).to.eql(thisProject.id);
+              cy.intercept('GET', awxAPI`/job_templates/${editedJt.id.toString()}/launch/`).as(
+                'clickLaunch'
+              );
+              cy.intercept('POST', awxAPI`/job_templates/${editedJt.id.toString()}/launch/`).as(
+                'launched'
+              );
+              cy.getByDataCy('launch-template').click();
+              cy.wait('@clickLaunch');
+              cy.wait('@launched')
+                .its('response.body')
+                .then((launched: Job) => {
+                  cy.verifyPageTitle(editedJt.name);
+                  cy.url().should('include', '/output');
+                  cy.get('[data-cy="relaunch-job"]').should('be.visible');
+                  cy.waitForJobToProcessEvents(launched.id.toString());
+                });
+            });
+          cy.navigateTo('awx', 'projects');
+          cy.verifyPageTitle('Projects');
+          cy.filterTableByMultiSelect('name', [thisProject.name]);
+          cy.get(`[data-cy="row-id-${thisProject.id}"]`).within(() => {
+            cy.get('[data-cy="name-column-cell"]').click();
+          });
+          cy.get(`a[href*="/projects/${thisProject.id}/job_templates?"]`).click();
```

Comment:
suggestion to use `cy.navigate`

Reply:
Yes, it was clicking the Job Templates tab, good suggestion. Thank you.